app/models/category.rb

```diff
@@ -24,6 +24,17 @@ class Category < VersionedRecord
   validate :sub_relation
   validate :only_manager_and_admin_users_can_be_assigned, if: :manager_id_changed?
 
+  def has_reporting_cycle_taxonomy?
+    Taxonomy.current_reporting_cycle_id == taxonomy_id
+  end
+
+  def is_current
+    has_reporting_cycle_taxonomy? &&
+      date.present? &&
+      category.present? &&
+      category.categories.order(date: :desc).first == self
+  end
```

Comment:
@parndt realising that we also need to exclude draft categories here (the latest, non-draft category from all its siblings (same parent) is the current category)

Reply:
[f0f147f](https://github.com/impactoss/impactoss-server/pull/406/commits/f0f147f1ba4fbd9263a329a829ae965979fa58e8)

Reply:
ah nice - that makes sense.